Makefile.am

```diff
@@ -152,7 +152,9 @@ check_DATA = tests/man.test
 # don't care. But if you are, then you need to run the tests anyway.
 tests/man.test tests/manonig.test: $(srcdir)/docs/content/manual/manual.yml
 if ENABLE_DOCS
-	$(AM_V_GEN) ( cd ${abs_srcdir}/docs; $(PIPENV) run python build_mantests.py )
+	$(AM_V_GEN) ( cd ${abs_srcdir}/docs; \
+		$(PIPENV) run python validate_manual_schema.py content/manual/manual.yml; \
+		$(PIPENV) run python build_mantests.py )
```

Comment:
Use .. && … to fail on validate error? Maybe already does somehow?

Reply:
Yeah, should be `&&`. Thanks.